src/azure-cli/HISTORY.rst

```diff
@@ -60,6 +60,10 @@ Release History
 * Added support for Backing up OS Disks only
 * Added restore-as-unmanaged-disk parameter to specify unmanaged restore
 
+**Batch**
+* az batch pool create: Added support to specify --disk-encryption-targets
+* Update Python SDK dependency to 9.0
```

Comment:
It is no longer required to add history notes here.
Instead, put it in PR title or add them to description of PR. Azure CLI will generate history notes automatically when releasing a new version.